### PR TITLE
Adjustments for sending Profile Name

### PR DESCRIPTION
**Pull request by Marthijn-B**

### CODE REVIEW

File: static/src/actions/auth.js [hunk 1]
```diff
@@ -120,10 +120,10 @@ export function registerUserFailure(error) {
     };
 }
 
-export function registerUser(email, password) {
+export function registerUser(profile, email, password) {
     return function (dispatch) {
         dispatch(registerUserRequest());
-        return create_user(email, password)
+        return create_user(profile, email, password)
```
Comment (mmdonaldson) [inline static/src/actions/auth.js:126]: You'll also need to update the create_user function which is inside utils/http_functions.js. Not sure why it's where, from the original boilerplate...

Comment (mmdonaldson) [reply static/src/actions/auth.js:126]: Ooops no worries you did this

**Review by mmdonaldson — DISMISSED** [review]

File: static/src/components/RegisterView.js [hunk 2]
```diff
@@ -108,8 +107,13 @@ export default class RegisterView extends React.Component {
     }
 
     login(e) {
+        console.log(this.state.profile);
```
Comment (mmdonaldson) [inline static/src/components/RegisterView.js]: Can we try and keep console.logs out of master now

Comment (Marthijn-B) [reply static/src/components/RegisterView.js]: Yep. saw your last post. all good

**Review by mmdonaldson — APPROVED** [review]

**Pull request merged**